Sources/Widgets/Manipulators/PickerManipulator/index.d.ts

```diff
@@ -0,0 +1,37 @@
+import { IAbstractManipulatorInitialValues, vtkAbstractManipulator } from "../AbstractManipulator";
+
+/**
+ *
+ */
+export interface IPickerManipulatorInitialValues extends IAbstractManipulatorInitialValues {
+
+}
+
+export interface vtkPickerManipulator extends vtkAbstractManipulator {
+
```

Comment:
We need to add the `setPicker` and `getPicker` types here, taking/returning `vtkPicker`.

Reply:
Changed in [8f4f0af](https://github.com/Kitware/vtk-js/pull/2907/commits/8f4f0af6ae4693be1f11023f8b00d50bded59731)